x-pack/plugins/reporting/server/browsers/chromium/paths.ts

```diff
@@ -16,44 +16,62 @@ interface PackageInfo {
   binaryRelativePath: string;
 }
 
-// We download zip files from a Kibana team GCS bucket named `headless_shell`
 enum BaseUrl {
+  // see https://www.chromium.org/getting-involved/download-chromium
+  common = 'https://commondatastorage.googleapis.com/chromium-browser-snapshots',
+  // A GCS bucket under the Kibana team
   custom = 'https://storage.googleapis.com/headless_shell',
 }
 
+interface CustomPackageInfo extends PackageInfo {
+  location: 'custom';
+}
+interface CommonPackageInfo extends PackageInfo {
+  location: 'common';
+  archivePath: string;
+}
+
 export class ChromiumArchivePaths {
-  public readonly packages: PackageInfo[] = [
+  public readonly revision = '856583';
+
+  public readonly packages: Array<CustomPackageInfo | CommonPackageInfo> = [
     {
       platform: 'darwin',
       architecture: 'x64',
-      archiveFilename: 'chromium-ef768c9-darwin_x64.zip',
-      archiveChecksum: 'd87287f6b2159cff7c64babac873cc73',
-      binaryChecksum: '8d777b3380a654e2730fc36afbfb11e1',
-      binaryRelativePath: 'headless_shell-darwin_x64/headless_shell',
+      archiveFilename: 'chrome-mac.zip',
+      archiveChecksum: '6aad6fa5a26d83e24e2f0d52de5230bf',
+      binaryChecksum: '2dc7a7250d849df4cab60f3b4a70c1ea',
+      binaryRelativePath: 'chrome-mac/Chromium.app/Contents/MacOS/Chromium',
+      location: 'common',
+      archivePath: 'Mac',
     },
     {
       platform: 'linux',
       architecture: 'x64',
-      archiveFilename: 'chromium-ef768c9-linux_x64.zip',
-      archiveChecksum: '85575e8fd56849f4de5e3584e05712c0',
-      binaryChecksum: '38c4d849c17683def1283d7e5aa56fe9',
+      archiveFilename: 'chromium-d163fd7-linux_x64.zip',
+      archiveChecksum: 'fba0a240d409228a3494aef415c300fc',
+      binaryChecksum: '99cfab472d516038b94ef86649e52871',
       binaryRelativePath: 'headless_shell-linux_x64/headless_shell',
+      location: 'custom',
     },
     {
       platform: 'linux',
       architecture: 'arm64',
-      archiveFilename: 'chromium-ef768c9-linux_arm64.zip',
-      archiveChecksum: '20b09b70476bea76a276c583bf72eac7',
+      archiveFilename: 'chromium-d163fd7-linux_arm64.zip',
+      archiveChecksum: '29834735bc2f0e0d9134c33bc0580fb6',
       binaryChecksum: 'dcfd277800c1a5c7d566c445cbdc225c',
```

Comment:
Why didn't this checksum change? 🤔

Reply:
Good catch! That seems to be an error in this code. Working on a fix...